docs/source/engine.rst

```diff
@@ -82,8 +82,9 @@ store the trainer and then resume the training. For example:
     optimizer = ...
     lr_scheduler = ...
     data_loader = ...
+    metric = ...
 
-    to_save = {'trainer': trainer, 'model': model, 'optimizer': optimizer, 'lr_scheduler': lr_scheduler}
+    to_save = {'trainer': trainer, 'model': model, 'optimizer': optimizer, 'lr_scheduler': lr_scheduler, 'metric': metric}
```

Comment:
Ideally, we would like to get that in a more automatic way. Also if we want to save multiple metrics adding them one by one to save/load can be a pain.

Reply:
Could you please reiterate what you mean? If we have multiple metrics, is it cumbersome to do:
```python
to_save = {..., 'metric1': metric1, 'metric2': metric2, 'metric3': metric3}
```
?

Reply:
Yes, it is too verbose, let's try to find a way to simplify that.